markdown_it/tree.py

```diff
@@ -61,6 +61,25 @@ def __init__(self) -> None:
         # children (i.e. inline or img)
         self._children: list = []
 
+        if create_root:
+            self._set_children_from_tokens(tokens)
+            return
+
+        if not tokens:
+            raise ValueError(
+                "Can only create root from empty token sequence."
+                " Set `create_root=True`."
+            )
+        elif len(tokens) == 1:
+            inline_token = tokens[0]
+            assert not inline_token.nesting
```

Comment:
maybe add an assertion message here, why this is not allowed

Reply:
Fixed